packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -39,6 +39,23 @@ export default declare((api, options) => {
         if (parentPath.isUnaryExpression({ operator: "delete" })) {
           replacementPath = parentPath;
         }
+
+        if (
+          falsyExpressionOptimisation &&
+          loose &&
+          parentPath.isIfStatement()
+        ) {
+          if (hasNoCallExpressionInChain(path)) {
```

Comment:
Why should we bail out in case of a function call? This seems ok:

```js
a?.b.c() 

// -> 

a && a.b.c()
```

Reply:
🤔 so the function call can only happened at the end right? because middle of the chain could be problematic:

```
a?.b()?.c

// ->

a && a.b() && a.b().c
```